bn-apps/memberships-management/membership-service/src/test/kotlin/net/corda/businessnetworks/membership/ActivateMembershipFlowTest.kt

```diff
@@ -47,6 +55,31 @@ class ActivateMembershipFlowTest : AbstractFlowTest(2) {
         assert(notification.second is OnMembershipActivated)
     }
 
+    @Test
+    fun `membership activation should succeed when using convenience flow`() {
```

Comment:
Should we extract this to a separate test, as its effectively testing a different flow?

Reply:
yeah, it's kinda the same flow I thought just different entrypoint to it but can do

Reply:
i think it would be better to keep this in the same test class given it all relates to activating memberships. Then we don't end up with that many test classes, which makes it hard to orientate in.

Reply:
I just want to make sure that we keep unit tests loosely scoped, without testig a bunch of different functionalities within the same class.